tests/pytorch_tests/model_tests/feature_models/const_representation_test.py

```diff
@@ -154,3 +154,56 @@ def __init__(self, unit_test, func, const, indices):
 
     def create_networks(self):
         return ConstRepresentationGetIndexNet(self.func, self.const, self.indices)
+
+
+class ConstRepresentationCodeNet(nn.Module):
+    def __init__(self, input_shape):
+        super().__init__()
+        self.input_shape = input_shape
+        self.conv2d = nn.Conv2d(3, 16, 3, 2, padding=1)
+        self.bn = nn.BatchNorm2d(16)
+        self.register_buffer('sub_const', 10 * torch.rand((1, 16, 64)))
+
+    def forward(self, x):
+        _shape = x.shape[2:]
+        x = self.conv2d(x)
+
+        # input tensor in kwargs
+        x = nn.functional.interpolate(x, size=_shape)
+
+        # reshaping batch_norm input to 3 axes to avoid bn-folding.
+        x = x.reshape((-1, 16, int(np.prod(self.input_shape))))
+
+        # input const in kwargs (not the first kwargs!)
+        x = nn.functional.batch_norm(x,
+                                     self.bn.running_mean, self.bn.running_var,
+                                     momentum=0.2, eps=1e-6, bias=self.bn.bias)
+
+        # input all tensors and consts in kwargs
+        x = torch.sub(input=self.sub_const, other=x)
```

Comment:
maybe also add to the model a functional op without key arguments? (just x+y or something like that)

Reply:
We have other specific tests for that.